pkg/question_generator/questions.go

```diff
@@ -0,0 +1,44 @@
+package question_generator
+
+import (
+	"github.com/crisBrunette/questioncicas/internal/database"
+)
+
+type Question struct {
+	UUID     int
+	Question string
+	Answer   string
+}
+
+type QuestionList struct {
+	QuestionList []Question
+}
+
+func (q Question) GetUUID() int {
+	return q.UUID
+}
+func (q Question) GetQuestion() string {
+	return q.Question
+}
+func (q Question) GetAnswer() string {
+	return q.Answer
+}
```

Comment:
In Go we usually don't create getters and setters, we simply use the field and that's it.
In some cases it makes sense, for example when you're creating a library that will be used by other unknown developers and you don't want them to directly modify your fields, but in general, this is not needed.
In fact in this case you've left the fields as public, and then created these getters which which are redundant, since now you can get the answer either by doing question.Answer or question.GetAnswer()